### PR TITLE
Update 14

### PR DESCRIPTION
**Pull request by Rainb0wKey**
#BDFD2023Wiki

### CODE REVIEW

File: src/bdscript/awaitFunc.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# $awaitFunc
+It's used to initiate an awaited command.
```
Comment (MineBartekSA) [inline src/bdscript/awaitFunc.md]:
```suggestion
Used to initiate an awaited command.
```

File: src/bdscript/awaitFunc.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Syntax
+```
+$awaitFunc[name;(user ID;channel ID)]
+```
+
+## Parameters
+- `name`: The name which is used inside [`$awaitedCommand[]`](../callbacks/awaitedCommand.md) and [`$awaitedCommandError[]`](../callbacks/awaitedCommandError.md) callback.
+- `user ID`: The user the awaited command will trigger for. Uses command author, if user ID is not given. _(Optional)_
+- `channel ID`: The channel where the command should be awaited. Uses current channel, if `channel ID` is not given. _(Optional)_
```
Comment (MineBartekSA) [inline src/bdscript/awaitFunc.md]:
```suggestion
- `name`: The name used inside [`$awaitedCommand[]`](../callbacks/awaitedCommand.md) and [`$awaitedCommandError[]`](../callbacks/awaitedCommandError.md) callbacks.
- `user ID`: The user the awaited command will trigger for. Uses command author, if `user ID` is not given. _(Optional)_
- `channel ID`: The channel where the command will be awaited. Uses current channel, if `channel ID` is not given. _(Optional)_
```

File: src/callbacks/awaitedCommand.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+_Triggered when an awaited command gets initiated._
+
+`$awaitedCommand[]` is a callback, which means it's used in the command trigger (not the code). The command is ran when an awaited command gets initiated.
+## Syntax
+```
+$awaitedCommand[name;(filter)]
+```
+## Parameters
+- `name`: The name which is used in [`$awaitFunc[]`](../bdscript/awaitFunc.md) function.
+- `name`: It is used to limit user input. _(Optional)_
```
Comment (MineBartekSA) [inline src/callbacks/awaitedCommand.md]:
```suggestion
- `name`: The name used in [`$awaitFunc[]`](../bdscript/awaitFunc.md) function.
- `filter`: It is used to limit user input. _(Optional)_

```

File: src/callbacks/awaitedCommand.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+### Supported filters
+- `<numeric>`: Accepts only number input.
+- `<word1/word2>`: Accepts only those specified words provided inside `<>`. Use `/` as separator for multiple words.
```
Comment (MineBartekSA) [inline src/callbacks/awaitedCommand.md]:
```suggestion
- `<word1/word2>`: Accepts only specified words provided inside `<>`. Use `/` as a separator for multiple words.
```

File: src/callbacks/awaitedCommandError.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+`$awaitedCommandError[]` is a callback, which means it's used in the command trigger (not the code). The command is ran when an awaited command doesn't match with provided filter.
+
+## Syntax
+```
+$awaitedCommandError[name]
+```
+
+## Parameters
+- `name`: The name which is used in [`$awaitFunc[]`](../bdscript/awaitFunc.md) function.
```
Comment (MineBartekSA) [inline src/callbacks/awaitedCommandError.md]:
```suggestion
- `name`: The name used in [`$awaitFunc[]`](../bdscript/awaitFunc.md) function.
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

**Comment by Rainb0wKey** [comment]
Ready^

**Review by MineBartekSA — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**